components/ImageSlider.tsx

```
@@ -0,0 +1,93 @@
+            style={{ zIndex: 1 }}
+          >
+            <IoChevronForward size={24} color="white" />
+          </button>
+        </>
+      )}
+
+      {images.length > 1 && (
+        <div
+          className="position-absolute bottom-0 start-50 translate-middle-x mb-2 bg-dark bg-opacity-50 px-2 py-1 rounded"
+          aria-live="polite"
+          role="status"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<output>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)